mycroft/skills/phillips_hue/test/intent/sample4.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "turn up the lights",
+  "intent_type": "IncreaseBrightnessIntent",
+  "intent": {
+    "IncreaseBrightnessKeyword": "turn up the lights"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab.